addon/components/au-data-table/text-search.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="au-c-data-table__search">
-  {{input value=value placeholder=placeholder class="au-c-input au-c-input--block"}}
+  <Input value={{@value}} placeholder={{this.placeholder}} class="au-c-input au-c-input--block" />
```

Comment:
Sorry, I might have been unclear with what I meant. It should be passed as an argument, not an attribute for the 2-way binding to work. It's the `Input` component that does the 2-way binding magic.
```suggestion
  <Input @value={{this.value}} placeholder={{this.placeholder}} class="au-c-input au-c-input--block" />
```

`@value` only returns the correct value here because in `@ember/components` there is no difference between component arguments and class properties.